python/F3DPythonBindings.cxx

```diff
@@ -290,13 +290,19 @@ PYBIND11_MODULE(pyf3d, module)
 
   py::enum_<f3d::window::Type>(window, "Type")
     .value("NONE", f3d::window::Type::NONE)
-    .value("NATIVE", f3d::window::Type::NATIVE)
-    .value("NATIVE_OFFSCREEN", f3d::window::Type::NATIVE_OFFSCREEN)
+    .value("AUTO", f3d::window::Type::AUTO)
     .value("EXTERNAL", f3d::window::Type::EXTERNAL)
+    .value("GLX", f3d::window::Type::GLX)
+    .value("WGL", f3d::window::Type::WGL)
+    .value("COCOA", f3d::window::Type::COCOA)
+    .value("EGL", f3d::window::Type::EGL)
+    .value("OSMESA", f3d::window::Type::OSMESA)
+    .value("UNKNOWN", f3d::window::Type::UNKNOWN)
```

Comment:
what does pybind11 does with this ?

Reply:
it's used when retrieving the window type.